sigle/src/modules/publicStory/PublicStory.tsx

```diff
@@ -74,10 +81,36 @@ export const PublicStory = ({ story, settings }: PublicStoryProps) => {
           "& :where(a strong):not(:where([class~='not-prose'] *))": {
             color: safeSiteColor,
           },
+
+          px: '$4',
+
+          '& p': {
+            m: 0,
+            p: 0,
+          },
         }}
       >
-        <h1 className="sigle-title">{story.title}</h1>
-        {story.coverImage && (
+        <Link href="/[username]" as={`/${username}`} passHref>
+          <a>
+            <Text size="action">{siteName}</Text>
+            <Box css={{ mt: '$1', mb: '$7' }}>
+              <Text
+                size="action"
+                css={{
+                  display: 'flex',
+                  gap: '$2',
+                  color: '$gray9',
+                }}
+              >
+                {format(story.createdAt, 'MMM dd')}
+                <span>•</span>
+                <span>{storyReadingTime?.text}</span>
+              </Text>
+            </Box>
+          </a>
+        </Link>
+        <Heading size="3xl">{story.title}</Heading>
```

Comment:
the current H1 should already have the proper styling, we can leave it that way :) (style is coming from tailwind)